UT-Alignment-12d-7/readme.md

```diff
@@ -0,0 +1,46 @@
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has an alignment with left arc in horizontal and crest and sag parabolas for vertical
```

Comment:
I assume this is copy-paste procreation of mistakes.
```suggestion
This scenario has an alignment with left arc in horizontal and crest and sag parabolas for vertical.
```

Reply:
Fixed in the RC3 version